guides/rails_guides/markdown/renderer.rb

```diff
@@ -90,6 +90,7 @@ def convert_notes(body)
           end
         end
 
+        # rubocop:disable Lint/EndAlignment
```

Comment:
Sure.. but if rubocop is insisting on something dumb, we should use its config to tell it to not do that (and hopefully, to instead insist on something less eye-bleedingly wrong).

Using a tool to enforce arbitrary style choices is one thing; having it impose secret work-arounds ("oh, it does that, but you can ignore it, and just do this instead..") is quite another.